cmd/livepeer/starter/starter.go

```diff
@@ -1227,10 +1227,27 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 					}
 				}
 
-				// If the config contains a URL we call Warm() anyway because AIWorker will just register
-				// the endpoint for an external container
 				if config.Warm || config.URL != "" {
+					// Register the external endpoint if URL is provided
 					endpoint := worker.RunnerEndpoint{URL: config.URL, Token: config.Token}
+
+					// If config.Warm is true, setup the AI worker with the selected Docker image
+					if config.Warm && config.URL == "" {
+						// Determine which image to use based on the pipeline
+						imageToUse := *cfg.AIRunnerImage // Default to the image from config
```

Comment:
@pschroedl can we replace this with a helperfunction like `getContainerImageID`?

Reply:
OK, SGTM.